extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -103,7 +135,13 @@ function measureTargetElement(state) {
  * @private
  */
 function showBubbleElement(state) {
-  state.bubbleElement.textContent = state.message;
+  removeChildren(state.bubbleElement);
+  const messageDiv = state.bubbleElement.ownerDocument.createElement('div');
+  messageDiv.id = state.bubbleElement.getAttribute('aria-labeledby');
+  messageDiv.textContent = state.message;
+  state.bubbleElement.setAttribute('role', 'alert');
```

Comment:
We do, the `state.targetElement` is basically the input. What would adding `aria-describedby` at this point would trigger?